content/asset-tracking/using-the-sdks.textile

```diff
@@ -343,6 +438,12 @@ subscriber.resolutionPreference(Resolution(Accuracy.MAXIMUM, desiredInterval = 1
 ...
 ```
 
+```[swift]
+subscriber.resolutionPreference(Resolution(accuracy: .low,
```

Comment:
The `resolutionPreference` function takes 2 arguments, and the completion handler is required.

Example from `MapViewController.swift`:
```swift
        subscriber?.resolutionPreference(resolution: resolution) { [weak self] result in
            switch result {
            case .success:
                self?.currentResolution = resolution
                self?.updateResolutionLabel()
                logger.info("Updated resolution to: \(resolution)")
            case .failure(let error):
                let errorDescription = DescriptionsHelper.ResolutionStateHelper.getDescription(for: .changeError(error))
                self?.showErrorDialog(withMessage: errorDescription)
            }
        }
```